packages/@angular/cli/tasks/npm-install.ts

```diff
@@ -11,7 +12,7 @@ export default Task.extend({
       packageManager = 'npm';
     }
 
-    return new Promise(function(resolve, reject) {
+    return checkYarnOrCNPM().then(function () {
```

Comment:
Oh no, I did it again and brake everything, just tried to rebase from upstream (seems that something were broken when I did make a pull request) and now my branch history looks like

```
other commit
other commit
my commit
prev commit
```

@jiverson can you please help me figure out what is right way to update repo so pull request is ready to be merged?